src/main/java/org/example/hw21/task2/Person.java

```diff
@@ -0,0 +1,30 @@
+package org.example.hw21.task2;
+
+import lombok.Data;
+
+@Data
+
+public class Person implements Cloneable{
```

Comment:
Shallow/deep copy should be implemented in scope of
"clone" method overriding.

In clone method you should return Person object (deep copy, with creating inside new address based on address of old object)